packages/form-field/src/FormField/useFormFieldProps.ts

```diff
@@ -7,6 +7,8 @@ export interface FormFieldInputElementProps {
   'aria-describedby': string;
   'aria-labelledby'?: string;
   'aria-label'?: string;
+  'aria-disabled'?: boolean;
+  'aria-invalid'?: FormFieldProps['aria-invalid'];
```

Comment:
why is this one the only one using `FormFieldProps`?

Reply:
`aria-disabled` will always be a boolean but `aria-invalid` could have other values like `'grammar'` or `'spelling'`. I didn't think we wanted to be overly opinionated on this and force it to be a boolean